modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/actions/ActionsInfo.js

```diff
@@ -0,0 +1,342 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
+import ClayForm, {ClayInput, ClaySelect} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import PropTypes from 'prop-types';
+import React, {useContext, useEffect, useState} from 'react';
+
+import {DiagramBuilderContext} from '../../../../DiagramBuilderContext';
+import SidebarPanel from '../../SidebarPanel';
+import {limitValue, sortElements} from '../utils';
+
+const DEFAULT_LIMIT = 1;
+const MIN_PRIORITY = 1;
+
+let executionTypeOptions = [
+	{
+		label: Liferay.Language.get('on-entry'),
+		value: 'onEntry',
+	},
+	{
+		label: Liferay.Language.get('on-exit'),
+		value: 'onExit',
+	},
+];
+
+const ActionsInfo = ({identifier, index, sectionsLength, setSections}) => {
+	const {selectedItem, setSelectedItem} = useContext(DiagramBuilderContext);
+
+	const [description, setDescription] = useState('');
+	const [executionType, setExecutionType] = useState('');
+	const [name, setName] = useState('');
+	const [priority, setPriority] = useState(DEFAULT_LIMIT);
+	const [template, setTemplate] = useState('');
+
+	useEffect(() => {
+		if (selectedItem.type === 'task' && executionTypeOptions.length < 3) {
```

Comment:
@alinedoleron I think it could be better to check directly if there's an option with value 'onAssignment' instead of relying on the array length.